posts/2021/10/hpy-0.0.4.md

```diff
@@ -0,0 +1,219 @@
+<!--
+.. title: hpy 0.0.4: Third public release
+.. slug: hpy-0.0.4-third-public-release
+.. date: 2021-10-22 15:00:00 UTC
+.. author: fangerer
+.. tags:
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+HPy 0.0.4 is out! The third official HPy release comes with many new features
+and was again made available on PyPI (first time with binary wheels).
+
+Besides new API functions, the major highlights of this release are Python 3.10
+support, better support for native fields (HPyField) and global variables
+(HPyGlobal), 
+
+Great new also is that we are now able to provide two more non-trivial projects
+that have been (partially) migrated to HPy. This is,
+[Kiwisolver](https://github.com/hpyproject/kiwi-hpy/)
+and [Matplotlib](https://github.com/hpyproject/matplotlib-hpy/).
```

Comment:
```suggestion
Great news too is that we are now able to provide two more non-trivial projects
that have been (partially) migrated to HPy. This is,
[Kiwisolver](https://github.com/hpyproject/kiwi-hpy/)
and [Matplotlib](https://github.com/hpyproject/matplotlib-hpy/).
```

Reply:
I would also add `We are still cleaning these ports up and will write another blog post about the ports and open them for discussion with the project owners.`